manifests/charts/gateway/templates/deployment.yaml

```
@@ -105,3 +105,4 @@ spec:
       topologySpreadConstraints:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      terminationGracePeriodSeconds: {{ $.Values.terminationGracePeriodSeconds | default 30 }}
```

```suggestion
      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
```
nit: this default seems redundant to me. You have the value defined in the values.yaml, and k8s default value is 30s as well, should we remove this?

---

agree, I removed the dafault 30 from this PR.